### DISCUSSION TITLE
VR 180 Images SBS

### DISCUSSION THREAD

**Discussion by dimdieo** [thygate/stable-diffusion-webui-depthmap-script#138] [Ideas]
Hello!  Thanks for the script, great job!  I noticed an interesting feature that if you generate an image in SD with a token in the Fisheye prompt and then process it with your script, you can get an interesting result.  If you view images on a VR headset set to FishEye 180 SBS (I used DEOVR for Oculus Quest 2), you can get even more of an immersive experience and it looks more like VR 180 SBS images.  True, people there seem to be giants :).  It would be great if we could generate full VR 180 images, this will bring us even closer to full immersion in the generative VR world of stable diffusion.

**Comment by thygate** [comment]
Nice find.

Also check this video on the topic : https://www.youtube.com/watch?v=t-8I7EkIL8c
